libs/platform/src/lib/form/checkbox/checkbox.component.ts

```diff
@@ -70,102 +56,82 @@ export class CheckboxComponent extends BaseInput implements AfterViewInit {
     @Input()
     tristate = false;
 
-    /** when true indeterminate state can be selected */
     @Input()
-    tristateSelectable = true;
+    set isBinary(value: boolean) {
+        if (isDevMode()) {
+            console.warn(
+                '"isBinary" is deprecated and has no effect anymore \n' +
+                    'Checkbox is binary by default. Use "tristate" input if you need to have indeterminate checkbox.'
+            );
+        }
+    }
 
-    /** value for checkbox control */
+    // this is undesired to have "checked" input instead of "value"
+    // but it was done this way initially and we have to keep this in order to not break anything
+    /**
+     * value for checkbox control
+     */
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('checked')
     get value(): any {
         return this.getValue();
     }
     set value(selectValue: any) {
         this.setValue(selectValue);
     }
 
-    /** value for checkbox when selected */
-    @Input('value')
-    get checkboxValue(): any {
-        return this._checkboxValue;
-    }
-    set checkboxValue(selectValue: any) {
-        this._checkboxValue = selectValue;
-    }
-
-    /**
-     * true when checkbox is checked. used when checkbox created outside form
-     */
+    /** when true indeterminate state can be selected */
     @Input()
-    get checked(): boolean {
-        return this._checked;
-    }
-    set checked(value: boolean) {
-        if (value !== this.checked) {
-            this._checked = value;
-        }
-    }
+    tristateSelectable = false;
 
-    /**
-     * @deprecated
-     * set state of individual checkbox. Used by CBG to set checkbox states */
-    @Input()
-    get stateType(): ControlState {
+    /** value for checkbox control */
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('value')
```

Comment:
@platon-rov it's absolute mess here, but if you look closer, you'll see these changes were not presented in this PR, it shows as diff because of line refactoring and new comments. 

I've started to refactor this, but if we provide proper naming for the inputs, in a lot of places checkboxes will stop working properly without an error:
-`checked` (that stands for actual value) typed as `any`, because anything can be assigned for any checkbox state. The only thing I did here is merged internal `value` and `checked` variables as it didn't have any sense to keep both
- `value` input is used for `FdCheckboxValues. trueValue `, that is again `any`. 

Since both are `any`, we can't really swap the purposes of them to resolve the confusion..